src/rgw/rgw_lc.cc

```diff
@@ -238,8 +238,11 @@ int RGWLC::bucket_lc_prepare(int index)
       ret = cls_rgw_lc_set_entry(store->lc_pool_ctx, obj_names[index],  entry);
       if (ret < 0) {
         dout(0) << "RGWLC::bucket_lc_prepare() failed to set entry on " << obj_names[index] << dendl;
-        break;
```

Comment:
agreed that the current behavior is wrong - a break here wouldn't update the `marker`, so it would re-list the `entries` from the same position. but i tend to think that this should just return failure instead